README.md

```diff
@@ -62,6 +62,17 @@ multiple times using the `loop_count` option to produce large amounts of output:
 
 In the example above, the animals-data.json records will be read and emitted as records 3 times.
 
+meltano.yml example
+```yml
```

Comment:
```suggestion
```yml
# meltano.yml
```

Reply:
nice, even if it's a full meltano.yml example would you do that? Not a big deal :)